test/common/page_objects/search-page.js

```diff
@@ -22,6 +22,7 @@ class FindPage {
       summary: this.extractText('summary', context),
```

Comment:
class FindPage .... but actually called search-page.js  might be worth renaming for consistancy